### ISSUE TITLE
FHIRHelpers.ToConcept cannot resolve call to operator with signature (System.Concept).

### ISSUE THREAD

**Issue by JuliaDawson** [cqframework/ecqm-content-qicore-2022#9]
Please see the attached zip file for screenshots, CQL and testcase for problems in STK-2.
[STK-2_ForToConcept_Error.zip](https://github.com/cqframework/ecqm-content-qicore-2022/files/9201191/STK-2_ForToConcept_Error.zip)

Error says 'Could not resolve call to operator ToConcept with signature (System.Concept).' in logic like below:

define "Pharmacological Contraindications for Antithrombotic Therapy at Discharge":
  ["MedicationRequest": medication in "Pharmacological Contraindications For Antithrombotic Therapy"] Pharmacological
      where Pharmacological.doNotPerform is true
        and exists ( Pharmacological.category C 
          where FHIRHelpers.ToConcept ( C ) ~ FHIRCommon."Community"
             or FHIRHelpers.ToConcept ( C ) ~ FHIRCommon."Discharge"
            and Pharmacological.status in { 'active', 'completed' }
            and Pharmacological.intent = 'order')

**Comment by drthyl** [comment]
Would the error resolve if you remove FHIRHelpers.ToConcept?
```
define "Pharmacological Contraindications for Antithrombotic Therapy at Discharge":
["MedicationRequest": medication in "Pharmacological Contraindications For Antithrombotic Therapy"] Pharmacological
where Pharmacological.doNotPerform is true
and exists ( Pharmacological.category C
where C ~ FHIRCommon."Community"
or C ~ FHIRCommon."Discharge"
and Pharmacological.status in { 'active', 'completed' }
and Pharmacological.intent = 'order')
```

**Comment by JuliaDawson** [comment]
It does! Thank you!

**Comment by brynrhodes** [comment]
Fixed in 0.4.0
